ingestion/src/metadata/orm_profiler/orm/types/bytea_to_string.py

```diff
@@ -49,4 +55,15 @@ def process_result_value(self, value: str, dialect) -> Optional[str]:
         if not value:
             return None
         self.validate(value)
+
+        try:
+            bytes_value = bytes(value)
+            detected_encoding = chardet.detect(bytes_value).get("encoding")
+            if detected_encoding:
+                value = bytes_value.decode(encoding=detected_encoding)
+                return value
+        except Exception as exc:
+            logger.debug(traceback.format_exc())
+            logger.error(exc)
+
```

Comment:
I think we should add the try except only for this operation `value = bytes_value.decode(encoding=detected_encoding)`. I believe the other ones won't throw an error (if i am not mistaking).

Also in `self.validate` let's add `if not isinstance(value, (memoryview, bytes, bytearray)):`

Reply:
Done